src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -325,12 +325,12 @@ private DetectProperties() {
             .setGroups(DetectGroup.CONTAINER_SCANNER, DetectGroup.SOURCE_PATH)
             .build();
 
-    public static final NullableStringProperty DETECT_RL_SCAN_FILE =
-            NullableStringProperty.newBuilder("detect.rl.scan.file.path")
-                .setInfo("ReversingLabs Scan Target", DetectPropertyFromVersion.VERSION_9_6_0)
+    public static final NullableStringProperty DETECT_THREAT_INTEL_SCAN_FILE =
+            NullableStringProperty.newBuilder("detect.threatintel.scan.file.path")
+                .setInfo("ThreatIntel Scan Target", DetectPropertyFromVersion.VERSION_9_6_0)
                 .setHelp(
-                    "If specified, this file and this file only will be uploaded for ReversingLabs analysis.  The RL_SCAN tool does not provide project and version name defaults to Detect, so you need to set project and version names via properties when only the RL_SCAN tool is invoked.")
-                .setGroups(DetectGroup.RL_SCANNER, DetectGroup.SOURCE_PATH)
+                    "If specified, this file and this file only will be uploaded for ThreatIntel analysis.  The THREAT_INTEL tool does not provide project and version name defaults to Detect, so you need to set project and version names via properties when only the THREAT_INTEL tool is invoked.")
```

Comment:
Joe, in the UI, uses "Threat Intel" with a space -- maybe let's go with that for consistency sake? (Just for the description, for the property name threatintel as one word is fine)